src/api/v3/feedback/__init__.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.http import HttpResponse
+
+
+def feedback_handler(func):
+    def handle_feedback(request, region_slug, language_code):
+        if request.method != 'POST':
+            return HttpResponse('Invalid request.', status=405)
+        data = json.loads(request.body)
+
+        comment = data.pop("comment", "")
+        rating = data.pop("rating", None)
+        category = data.pop("category", None)
+
+        if rating and rating != 'up' and rating != 'down':
+            return HttpResponse("Invalid rating.", status=400)
+        if comment == "" and not rating:
+            return HttpResponse("Either comment or rating is required.", status=400)
+        if rating == 'up':
+            emotion = 'Pos'
+        elif rating == 'down':
+            emotion = 'Neg'
+        else:
+            emotion = "NA"
+        is_technical = category == "Technisches Feedback" #todo: determine is_technical
```

Comment:
I clarified with the app team that this is a long-term legacy convention (so nothing else to do).
```suggestion
        is_technical = category == "Technisches Feedback"
```